Library/brew.sh

```diff
@@ -13,6 +13,10 @@ odie() {
   else
     echo "$*" >&2
   fi
+}
+
+odie() {
+  odie "$*"
```

Comment:
- This should be calling `onoe` instead of itself.
- Please make that `onoe "$@"` instead. It's going to misbehave otherwise if `odie` is called as `odie <<EOS`. (`"$*"` joins all arguments into a single argument, but it will always pass one argument. `"$@"` transparently forwards all arguments as individual arguments.)